gui/public/i18n/en/translation.ftl

```diff
@@ -1152,12 +1152,10 @@ firmware_tool-flash_method_step-serial =
 firmware_tool-flashbtn_step = Press the boot btn
 firmware_tool-flashbtn_step-description = Before going into the next step there is a few things you need to do
 
-firmware_tool-flashbtn_step-board_SLIMEVR = Press the flash button on the pcb before inserting turning on the tracker.
-    If the tracker was already on, simply turn it off and back on while pressing the button or shorting the flash pads.
-    Here are a few pictures on how to do it according to the different revisions of the SlimeVR tracker
-firmware_tool-flashbtn_step-board_SLIMEVR-r11 = Turn on the tracker while shorting the second rectangular FLASH pad from the edge on the top side of the board, and the metal shield of the microcontroller
-firmware_tool-flashbtn_step-board_SLIMEVR-r12 = Turn on the tracker while shorting the circular FLASH pad on the top side of the board, and the metal shield of the microcontroller
-firmware_tool-flashbtn_step-board_SLIMEVR-r14 = Turn on the tracker while pushing in the FLASH button on the top side of the board
+firmware_tool-flashbtn_step-board_SLIMEVR = Turn off the tracker, remove the case (if any), connect a USB cable to this computer, then do one of the following steps according to your SlimeVR board revision:
```

Comment:
i am really not a fan of this change. the word ``while`` vs ``then`` does not mean the same thing and is prone to user error.

then: leaves room for users shorting the pad -> releasing -> then -> turn on tracker
while: short the pad -> while -> turning on the tracker.